internal/pkg/dependencies/common.go

```diff
@@ -187,7 +188,7 @@ func (v *CommonContainer) Template(reference model.TemplateRef) (*template.Templ
 	// Get template
 	templateRecord, found := repo.GetTemplateById(reference.TemplateId())
 	if !found {
-		return nil, err
+		return nil, manifest.TemplateNotFoundError{}
```

Comment:
Uff, odhalil jsem díky e2e testu že detail verze háže 500ku při špatně zadané templatě, ale dalo mi pořádně zabrat než jsem to tady našel. 🙂